// text-chart/dist/interfaces.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
var ChartType;
(function (ChartType) {
    ChartType["Bar"] = "bar";
    ChartType["Histogram"] = "histogram";
})(ChartType = exports.ChartType || (exports.ChartType = {}));
var BarCharacters;
(function (BarCharacters) {
    BarCharacters["LightShade"] = "\u2591";
    BarCharacters["MediumShade"] = "\u2592";
    BarCharacters["DarkShade"] = "\u2593";
    BarCharacters["BlackSquare"] = "\u25A0";
    BarCharacters["WhiteSquare"] = "\u25A1";
    BarCharacters["BlackRectangle"] = "\u25AE";
    BarCharacters["WhiteRectangle"] = "\u25AF";
    BarCharacters["HorizontalFillSquare"] = "\u25A4";
    BarCharacters["VerticalFillSquare"] = "\u25A5";
    BarCharacters["OrthogonalCrosshatchFillSquare"] = "\u25A6";
    BarCharacters["UpperLeftToLowerRightFillSquare"] = "\u25A7";
    BarCharacters["UpperRightToLowerLeftFillSquare"] = "\u25A8";
    BarCharacters["DiagonalCrosshatchFillSquare"] = "\u25A9";
    // Add on as desired
})(BarCharacters = exports.BarCharacters || (exports.BarCharacters = {}));
